src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlUtil.cs

```diff
@@ -416,26 +416,30 @@ internal static void SetTimeoutException(TaskCompletionSource<object> completion
                 );
             }
         }
-
-#if NET
-        internal static void SetTimeoutExceptionWithState(TaskCompletionSource<object> completion, int timeout, object state, Func<object, Exception> onFailure, CancellationToken cancellationToken)
+        
+        internal static void SetTimeoutExceptionWithState(
```

Comment:
By adding this project to the netfx project, we can use the slightly more performant "with state" version of this method.